l10n_br_account_product/test/account_customer_invoice.yml

```diff
@@ -80,6 +80,34 @@
   !assert {model: account.invoice, id: account_invoice_customer_sp_sp}:
     - state == 'sefaz_export'
 -
+  I check invoice totals
+-
+  !assert {model: account.invoice, id: account_invoice_customer_sp_sp}:
```

Comment:
:+1: But need add check for fields in account.invoice: icms_base, icms_value and others taxes....